app/adapters/threema_adapter/inbound.rb

```diff
@@ -31,9 +31,11 @@ def delivery_receipt?(decrypted_message)
     end
 
     def initialize_text(decrypted_message)
-      return nil unless decrypted_message.instance_of? Threema::Receive::Text
-
-      decrypted_message.content
+      if decrypted_message.instance_of? Threema::Receive::Text
+        decrypted_message.content
+      elsif decrypted_message.instance_of?(Threema::Receive::File) && decrypted_message.caption
+        decrypted_message.caption
+      end
```

Comment:
perhaps add an explicit return value of nil if none of the cases match
```suggestion
      end
      return nil
```